pkg/controller/networkpolicy/validate.go

```diff
@@ -36,17 +65,45 @@ var (
 	// reservedTierNames stores the set of Tier names which cannot be deleted
 	// since they are created by Antrea.
 	reservedTierNames = sets.NewString("baseline", "application", "platform", "networkops", "securityops", "emergency")
+	// apv is an instance of antreaPolicyValidator to validate Antrea-native
+	// policy events.
+	apv = antreaPolicyValidator{}
+	// tv is an instance of tierValidator to validate Tier resource events.
+	tv = tierValidator{}
```

Comment:
is there a reason for making these global `var`s instead of instantiating them in `NewNetworkPolicyValidator` and just adding them to `antreaPolicyValidators` / `tierValidators`?

Reply:
done